.rubocop.yml

```diff
@@ -0,0 +1,26 @@
+---
```

Comment:
I tried that initially, but it's incompatible with Ruby 1.8 and Puppet's vendor-bundled SafeYAML module.  I've added it again with some conditional logic restricting it to Ruby 2.2 and newer.  It's not as clean as the rest of the checks, but it passes the latest build.